### PR TITLE
update deployment credentials for new repo location

### PR DESCRIPTION
**Pull request by mattbennett**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,10 +47,10 @@ script:
 
 deploy:
   provider: pypi
-  user: onefinestay
+  user: mattbennett
```
Comment (davidszotten) [inline .travis.yml]: not sure we want this

Comment (davidszotten) [inline .travis.yml]: i would be happy to keep ofs until we have an issue, and at that point make a nameko one

Comment (davidszotten) [inline .travis.yml]: see also #381

**Comment by mattbennett** [comment]
Closing in favour of #381

**Pull request closed without merging**